src/app/ui/sequence.tsx

```diff
@@ -504,6 +491,14 @@ export class SequenceView extends PluginUIComponent<Props, SequenceViewState> {
 
                 if (!this.entityChainPairs) return;
 
+                const firstItem = this.entityChainPairs.chainOptions[0];
+                if (
+                    this.entityChainPairs.chainOptions.length === 1 &&
```

Comment:
`this.entityChainPairs.chainOptions` used twice, local const.

Reply:
Yes, it is a bit weird, but it is because while the sequence is loading, it will have just one item with entityId like `[""]`. I don't know why, but at least I can use that to know when the sequence hasn't loaded yet.

Reply:
Ok, let's keep it.